v21.2/restore.md

```diff
@@ -4,11 +4,7 @@ summary: Restore your CockroachDB cluster to a cloud storage services such as AW
 toc: true
 ---
 
-{{site.data.alerts.callout_info}}
- `RESTORE` no longer requires an Enterprise license, regardless of the options passed to it or to the backup it is restoring.
-{{site.data.alerts.end}}
-
-The `RESTORE` [statement](sql-statements.html) restores your cluster's schemas and data from [a `BACKUP`](backup.html) stored on a services such as AWS S3, Google Cloud Storage, NFS, or HTTP storage.
+The `RESTORE` [statement](sql-statements.html) restores your cluster's schemas and data from [a `BACKUP`](backup.html) stored on services such as AWS S3, Google Cloud Storage, NFS, or HTTP storage.
```

Comment:
same comments as v21.1, but one thing to note (maybe this gets updated for patch releases), but users can now use the LATEST keyword for SHOW BACKUP and RESTORE https://github.com/cockroachdb/cockroach/pull/73454

Reply:
I think for this one, I will wait for the generated docs issue to come through and prioritize it within the next few weeks, otherwise this PR is going to get quite large. Thanks for bringing this to my attention though!

Reply:
makes sense!

Reply:
@kathancox  - agreed we can do it in another PR. The change for "restore from latest" was merged a while back, so it should be in the backlog of docs issues. We can look for it at Triage on Monday.